about.html

```diff
@@ -0,0 +1,84 @@
+          <a href="https://twitter.com/microverse" target="_blank" rel="noopener noreferrer" class="fa fa-twitter"></a>
+          <a href="#English" target="_blank" rel="noopener">English</a>
+          <a href="#MyPage" target="_blank" rel="noopener">My Page</a>
+          <a href="#Logout" target="_blank" rel="noopener">Logout</a>
+        </div>
+      </section>
+      <section class="white-banner">
+        <div class="logo-container">
+          <a href="/" class="logo-container-2">
+            <img src="ressources/event-image/EoW-Logo.png" alt="A red logo for the End of the World Event" class="logo">
+          </a>
+        </div>
+        <div class="menu-container">
+          <div id="overlay-screen" class="overlay">
+            <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+            <nav class="overlay-content menu">
+              <a href="./about.html" class="active">About</a>
+              <a href="#Program">Program</a>
+              <a href="#Jobs">Jobs</a>
+              <a href="#Sponsor">Sponsor</a>
+              <a href="#News">News</a>
+            </nav>
+          </div>
+          <span class="hamburger" onclick="openNav()">&#9776;</span>
+        </div>
+        <nav class="nav-menu">
+          <a href="./about.html" class="active">About</a>
+          <a href="#Program">Program</a>
+          <a href="#Jobs">Jobs</a>
+          <a href="#Sponsor">Sponsor</a>
+          <a href="#News">News</a>
+        </nav>
+        <div class="button-container">
+          <button class="header-button" type="button" name="button">CC Campaing</button>
+        </div>
+      </section>
+    </header>
+    <section class="grey-background">
+      <div class="about-text">
+        <h2 class="page-title">End of the World Event 2022</h2>
+        <p class="about-intro">The end is near, and the world is doing one last event to finish lives of 7 billions people this year. for the occasion amazing VIP will accompagny you along the way to leave all together. Would you rather be alone and bored, or dance like there is no tomorrow and have no regrets ?</p>
+        <h5>Please don't try to resist the event, it's the last thing you will do in your life</h5>
+        <h6 href = "mailto: endtogether@endoftheworld.com">endtogether@endoftheworld.com</h6>
+      </div>
+    </section>
+    <section>
+      <h3 class="title">Past Events</h3>
+      <hr>
+      <div class="past-event">
+        <div class="past-card past-card1">
+          <div class="orange-overlay">
+            <h4 class="date">- 13.8b</h2>
+            <p class="about-event">The Big Bang</p>
+          </div>
+        </div>
+        <div class="past-card past-card2">
+          <div class="orange-overlay">
+            <h4 class="date">- 65m</h2>
+            <p class="about-event">The Dinosaur Ending</p>
+          </div>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] You are missing the footer in your about page, please add it

Reply:
Also added on both page

Reply:
Well solved

Reply:
Added the footer, however, I would recommend adding `overflow-x: hidden` in your CSS to fix the overflow of the page.
![html2](https://user-images.githubusercontent.com/55635977/142189350-004dda10-3a8f-40aa-aa90-957ea432010f.png)

Reply:
`overflow-x: hidden` added on .footer on css